Files/Views/Pages/Properties.xaml.cs

```diff
@@ -54,16 +57,23 @@ private async void Properties_Loaded(object sender, RoutedEventArgs e)
                 if (selectedItem.PrimaryItemAttribute == StorageItemTypes.File)
                 {
                     selectedStorageItem = await StorageFile.GetFileFromPathAsync(selectedItem.ItemPath);
+                    ItemProperties.ItemSize = selectedItem.FileSize;
                 }
                 else if (selectedItem.PrimaryItemAttribute == StorageItemTypes.Folder)
                 {
-                    selectedStorageItem = await StorageFolder.GetFolderFromPathAsync(selectedItem.ItemPath);
+                    var storageFolder = await StorageFolder.GetFolderFromPathAsync(selectedItem.ItemPath);
```

Comment:
Folder size calculation should be async and shouldn't block Properties UI.

Reply:
Fixed in this [commit](https://github.com/files-community/files-uwp/commit/59fc0f8c336ca5a12a505f43f2071d2e9272990d)